src/lib/playlists.ts

```diff
@@ -50,7 +50,7 @@ export async function loadAllPlaylistsByPage(retryCount = 0): Promise<any> {
         playlists: {data, count},
       } = await request(config.graphQLEndpoint, query, {
         page: currentPage,
-        per_page: 25,
+        per_page: 200,
```

Comment:
loadAllPlaylistsByPage fetches 25 items per page sequentially. This causes many slow network roundtrips.